lib/features/connection-preview/ConnectionPreview.js

```diff
@@ -150,7 +150,9 @@ ConnectionPreview.prototype.drawPreview = function(context, canConnect, hints) {
     connection.waypoints = this._connectionDocking.getCroppedWaypoints(connection, source, target);
   }
 
-  this._graphicsFactory.drawConnection(connectionPreviewGfx, connection);
+  this._graphicsFactory.drawConnection(connectionPreviewGfx, connection, {
+    stroke: 'var(--element-dragger-color)'
```

Comment:
:heart:

Reply:
Alternatively we may want to draw it, attaching a class.

Reply:
Attaching a class is different. Passing a stroke color to the renderer makes sure the element is properly rendered. For example, the reason the dragger for gateways doesn't look right at the moment is because we're using a class and based on that just unset every `fill`.

![image](https://github.com/bpmn-io/diagram-js/assets/7633572/e5ea1478-657f-4203-9502-179936fd6e61)

Reply:
:+1: